docs/source/configuration_file.rst

```diff
@@ -447,6 +446,47 @@ See details in both ``train_validation`` and ``test`` for the contrasts that are
         }
     }
 
+
+.. jsonschema::
+
+    {
+        "$schema": "http://json-schema.org/draft-04/schema#",
+        "title": "patch_filter_params",
+        "description": "Discard a 2D patch from the dataset if it meets a condition at training time, see below.",
+        "type": "dict",
+        "options": {
+            "filter_empty_input": {
+                "type": "boolean",
+                "description": "Discard 2D patches where all voxel intensities are zeros.Default: ``False``."
+            },
+            "filter_empty_mask": {
+                "type": "boolean",
+                "description": "Discard 2D patches where all voxel labels are zeros.Default: ``False``."
+            },
+            "filter_absent_class": {
+                "type": "boolean",
+                "$$description": [
+                    "Discard 2D patches where all voxel labels are zero for one or more classes\n",
+                    "(this is most relevant for multi-class models that need GT for all classes).\n",
+                    "Default: ``False``."
+                ]
+            }
+        }
+    }
+
+
+.. code-block:: JSON
+
+    {
+        "loader_parameters": {
+            "patch_filter_params": {
+                "filter_empty_mask": false,
+                "filter_empty_input": false
+            }
```

Comment:
missing `filter_absent_class` here?

Reply:
`config_default.json` is used to "fill" missing config key with default values [here](https://github.com/ivadomed/ivadomed/blob/d6fe24db4d6a4319c57802b7a6d68e17f6d8b9e0/ivadomed/main.py#L582). In this case (and for slice filter as well), as long as there is at least one param in the config, it will work. I did the same as slice filter with the 2 most likely used filters.